commands.md

```diff
@@ -18,7 +18,11 @@
 | [podman-export(1)](/docs/podman-export.1.md)             | Export container's filesystem contents as a tar archive                   |[![...](/docs/play.png)](https://asciinema.org/a/913lBIRAg5hK8asyIhhkQVLtV)|
 | [podman-generate(1)](/docs/podman-generate.1.md)         | Generate structured output based on Podman containers and pods | |
 | [podman-history(1)](/docs/podman-history.1.md)           | Shows the history of an image                                             |[![...](/docs/play.png)](https://asciinema.org/a/bCvUQJ6DkxInMELZdc5DinNSx)|
-| [podman-image(1)](/docs/podman-image.1.md)             | Manage Images||
+| [podman-image(1)](/docs/podman-image.1.md)               | Manage Images||
+| [podman-image-exists(1)](/docs/podman-image-exists.1.md) | Returns and exist status whether or not the image exists||
```

Comment:
'and exists' needs to at least change to "a exist".  Perhaps "Returns a status indicating the existence of the image"

Reply:
Simpler might be better - "Check if an image exists"

Reply:
Fixed